src/wizard/apple/index.md

```diff
@@ -0,0 +1,50 @@
+---
```

Comment:
Make sure to update this in sentry's `platformCategories`

Reply:
https://github.com/getsentry/sentry/pull/20799